bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/MetadataBookieDriver.java

```diff
@@ -68,6 +71,36 @@ LedgerManagerFactory getLedgerManagerFactory()
      */
     LayoutManager getLayoutManager();
 
+    /**
+     * Return health check is enable or disable.
+     *
+     * @return true if health check is enable, otherwise false.
+     */
+    default CompletableFuture<Boolean>  isEnableHealthCheck() {
+        return FutureUtils.value(true);
+    }
+
+    /**
+     * Disable health check.
+     */
+    default CompletableFuture<Void> disableHealthCheck() {
+        return FutureUtils.Void();
+    }
+
+    /**
+     * Enable health check.
+     */
+    default CompletableFuture<Void>  enableHealthCheck() {
+        return FutureUtils.Void();
+    }
+
+    /**
+     * @return disable health check path
+     */
+    default String getEnableHealthPath() {
```

Comment:
this "path" looks like a ZK configuration and not an abstraction, I'd remove it from the interface

Reply:
You are right! I will remove getEnableHealthPath in MetadataBookieDriver